editor/grida-canvas/reducers/metadata.reducer.ts

```
@@ -0,0 +1,33 @@
+  switch (action.type) {
+    case "node-metadata/set":
+      return {
+        ...state,
+        [action.node_id]: {
+          ...state[action.node_id],
+          [action.namespace]: action.data,
+        },
+      };
+    case "node-metadata/remove":
+      const { [action.namespace]: removed, ...rest } =
+        state[action.node_id] || {};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)